src/wallet/walletReducer.js

```diff
@@ -18,10 +29,49 @@ export default function walletReducer(state = initialState, action) {
         ...state,
         transferVisible: false,
       };
+    case GET_GLOBAL_PROPERTIES.SUCCESS: {
+      const { total_vesting_shares, total_vesting_fund_steem } = action.payload;
```

Comment:
I don't think that we should use destructuring inside reducers when variables are only used once. This adds unnecessary block inside case.